addons/web/static/src/views/list/list_controller.xml

```diff
@@ -79,7 +79,7 @@
     </t>
 
     <t t-name="web.ListView.Buttons" owl="1">
-        <div t-if="props.showButtons" class="o_list_buttons d-flex d-empty-none align-items-baseline" role="toolbar" aria-label="Main actions">
+        <div t-if="props.showButtons" class="o_list_buttons d-flex d-empty-none align-items-baseline gap-1" role="toolbar" aria-label="Main actions">
```

Comment:
Hello @chgo-odoo , thanks. 
`gap-1` adds an undesired gap when no buttons are rendered (eg. To-do listView)
![image](https://github.com/odoo/odoo/assets/8558351/5e025ed4-6b8c-40ce-8fac-48a17ff1ccb4)

I tested briefly and seems like gap is not needed actually, could you investigate?

Reply:
Thanks for noticing it, just had a look and it can be fixed by changing the margin. On it 👍